### PR TITLE
update install.sh and configuration.nix

### PR DESCRIPTION
**Pull request by ifurther**
update sshd setup ssh-rsa for new openssh

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-packer_http=$(cat .packer_http)
+packer_http=$PACKER_HTTP_ADDR
```
Comment (ihgann) [inline scripts/install.sh:3]: Why not just use `$PACKER_HTTP_ADDR` later in the script?

Separately - this means the `boot_command` can remove writing to the `.packer_http` file, right?

File: scripts/postinstall.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ for x in $(seq 0 2) ; do
 done
 
 # Remove install ssh key
-rm -rf /root/.ssh /root/.packer_http
+rm -rf /nixos/.ssh /nixos/.packer_http
```
Comment (ihgann) [inline scripts/postinstall.sh:14]: No reason to write `.packer_http`, so this can be omitted too, right?

**Review by ihgann — CHANGES_REQUESTED** [review]

File: scripts/install.sh (hunk 1 above)
Comment (ifurther) [reply scripts/install.sh:3]: Sometimes $PACKER_HTTP_ADDR is not corrtect, I think use the packer_http to control it. I only test it in virtualbox, so I could upgrade virtualbox boot_command.

File: scripts/postinstall.sh (hunk 2 above)
Comment (ifurther) [reply scripts/postinstall.sh]: Yes, i remove it.

File: scripts/postinstall.sh [hunk 3]
```diff
@@ -11,7 +11,6 @@ for x in $(seq 0 2) ; do
 done
 
 # Remove install ssh key
```
Comment (zimbatm) [inline scripts/postinstall.sh:13]: also remove the comment?

**Pull request merged**